e2e/main.e2e.spec.ts

```diff
@@ -0,0 +1,83 @@
+import path from 'path';
+import { Application, SpectronClient } from 'spectron';
+import { BrowserWindow } from 'electron';
+import NavBarPage from './pages/navbar';
+import GeneralPage from './pages/general';
+import KubernetesPage from './pages/kubernetes';
+import PortForwardingPage from './pages/portforwarding';
+import ImagesPage from './pages/images';
+import TroubleshootingPage from './pages/troubleshooting';
+const electronPath = require('electron');
+
+jest.setTimeout(60_000);
+
+describe('Rancher Desktop', () => {
+  let app:Application;
+  let client: SpectronClient;
+  let browserWindow: BrowserWindow;
+  let navBarPage: NavBarPage;
+  
+  beforeAll(async() => {
+    app = new Application({
+      // 'any' typing is required for now as other alternate usage/import
+      //  cause issues running the tests. Without 'any' typescript
+      //  complains of type mismatch.
+      path: electronPath as any,
+      args: [path.dirname(__dirname)],
+      env:  { SPECTRON_RUN: 'yes' }
```

Comment:
This is not great, since the user may have that set in their environment for some reason (maybe they're developing a _different_ electron app…).  I guess we can handle it in a follow-up PR?

Reply:
Agreed. We can handle in a follow-up PR.

Reply:
Spun off into #446